doc/usage.rst

```diff
@@ -64,191 +63,41 @@ Config file:
-    >>> enc = preprocessing.OneHotEncoder(categorical_features=categorical)
-    >>> print(enc)
-    OneHotEncoder(categorical_features=[False, True, True, False, True, True, True, True, True],
-           dtype=<class 'numpy.float64'>, handle_unknown='error',
-           n_values='auto', sparse=True)
-    >>> X = enc.fit_transform(X).todense()
-    >>> clf = ensemble.RandomForestClassifier()
-    >>> clf.fit(X, y)
-    RandomForestClassifier(bootstrap=True, class_weight=None, criterion='gini',
-                max_depth=None, max_features='auto', max_leaf_nodes=None,
-                min_impurity_split=1e-07, min_samples_leaf=1,
-                min_samples_split=2, min_weight_fraction_leaf=0.0,
-                n_estimators=10, n_jobs=1, oob_score=False, random_state=None,
-                verbose=0, warm_start=False)
-
-When you have to retrieve several datasets, you can use the convenience function
-:meth:`openml.datasets.get_datasets()`, which downloads all datasets given by
-a list of IDs:
-
-    >>> ids = [12, 14, 16, 18, 20, 22]
-    >>> datasets = openml.datasets.get_datasets(ids)
-    >>> print(datasets[0].name)
-    mfeat-factors
+OpenML contains several key concepts which it needs to make machine learning
+research shareable. A machine learning experiment consists of several runs,
+which describe the performance of an algorithm (called a flow in OpenML) on a
+task. Task is the combination of a dataset, a split and an evaluation metric. In
+this user guide we will go through listing and exploring existing tasks to
+actually running machine learning algorithms on them. In a further user guide
+we will examine how to search through datasets in order to curate a list of
+tasks.
 
 ~~~~~~~~~~~~~~~~~~
 Working with tasks
 ~~~~~~~~~~~~~~~~~~
 
-#TODO put a link to the OpenML documentation here! Link the Task functions and
-the task class
-
-While datasets provide the most basic information for a machine learning task,
-they do not provide enough information for a reproducible machine learning
-experiment. A task defines how to split the dataset into a train and test set,
-whether to use several disjoint train and test splits (cross-validation) and
-whether this should be repeated several times. Also, the task defines a target
-metric for which a flow should be optimized.
+Tasks are containers, defining how to split the dataset into a train and test
+set, whether to use several disjoint train and test splits (cross-validation)
+and whether this should be repeated several times. Also, the task defines a
+target metric for which a flow should be optimized. You can think of a task as
+an experimentation protocol, describing how to apply a machine learning model
+to a dataset in a way that it is comparable with the results of others (more
+on how to do that further down).
 
-Just like datasets, tasks are identified by IDs and can be accessed in three
-different ways:
+Tasks are identified by IDs and can be accessed in two different ways:
```

Comment:
Can we not filter by tags? Maybe I would say "you can explore tasks on the website or via ``list_tasks``. You can get a single task with ``get_task``". Because these two methods do semantically very different things.